devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy

```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright 2018 SmartThings
```

Comment:
The copyright date should not be changed

Reply:
Actually that's a 'fix', I've created this DTH last year, but apparently copied copyright statement without noticing old date.

Reply:
@tpmanley Soooo, should I change it back to 2018?

Reply:
You can leave it